geometry/include/pcl/geometry/mesh_indices.h

```diff
@@ -183,20 +180,29 @@ operator>>(std::istream& is, MeshIndex<MeshIndexTag>& index)
   return (is >> index.index_);
 }
 
-using VertexIndex = MeshIndex<struct VertexIndexTag>;
-using HalfEdgeIndex = MeshIndex<struct HalfEdgeIndexTag>;
-using EdgeIndex = MeshIndex<struct EdgeIndexTag>;
-using FaceIndex = MeshIndex<struct FaceIndexTag>;
-
 } // End namespace detail
 } // End namespace pcl
 
+namespace pcl {
+namespace geometry {
+/** \brief Index used to access elements in the half-edge mesh. It is basically
+ * just a wrapper around an integer with a few added methods. \author Martin
```

Comment:
Could you just  copy paste the doxygen from the original version here and before the other `using` statements?